kie-api/src/main/java/org/kie/api/builder/KieBuilder.java

```diff
@@ -39,6 +41,13 @@
     KieBuilder buildAll();
 
     /**
+     * Builds all the KieBases contained in the KieModule for which this KieBuilder has been created
+     * @param classFilters Used to prevent compilation of Java source files.
+     *          A Java source file vetoed by any of the given filters will not be compiled.
+     */
+    KieBuilder buildAll(Collection<KieClassFilter> classFilters);
```

Comment:
@mariofusco I hadn't considered the InternalKieBuilder as an option, but now that you have suggested it, I think that makes the most sense.